src/main.ts

```diff
@@ -110,19 +110,21 @@ async function main() {
         // OIDC specific checks
         if (enableOIDC) {
             console.log('Using OIDC authentication...')
-            try {
-                //generating ID-token
-                let audience = core.getInput('audience', { required: false });
+            //generating ID-token
+            let audience = core.getInput('audience', { required: false });
+            try{
```

Comment:
Code style is important.

```suggestion
            try {
```

Read more from https://www.google.com.hk/search?q=why+is+code+style+important